ocean_lib/assets/test/test_asset_downloader.py

```diff
@@ -171,9 +172,10 @@ def test_ocean_assets_download_destination_file(
         consumer=publisher_wallet.address,
         service_index=ddo.get_index_of_service(access_service),
         provider_fees=provider_fees,
-        consume_market_order_fee_address=publisher_wallet.address,
-        consume_market_order_fee_token=datatoken.address,
-        consume_market_order_fee_amount=0,
+        consume_market_fees=TokenFeeInfo(
+            address=publisher_wallet.address,
+            token=datatoken.address,
+        ),
```

Comment:
I recommend constructing `consume_market_fees = FeeTokenInfo(..)` right above the call to `datatoken.start_order()`. It will make the method call easier to read.